Retail/main.tf

```diff
@@ -0,0 +1,471 @@
+# retail/main.tf
```

Comment:
Please apply similar fixes to retail as well (similar to PSFT).
Retail whitepaper here: https://cloud.oracle.com/iaas/whitepapers/deploy_oracle_retail_merchandising_suite_across_oci_azure.pdf